packages/snf_mcp/snf_mcp.0.2.1/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis:
+  "A Model Context Protocol server for web search and content fetching"
+description:
+  "A Model Context Protocol (MCP) server that provides web search and web content fetching capabilities, written in OCaml using the Eio asynchronous runtime. Includes rate limiting and full MCP protocol compatibility."
+maintainer: ["Marcello Seri <marcello.seri@gmail.com>"]
+authors: ["Marcello Seri <marcello.seri@gmail.com>"]
+license: "ISC"
+tags: ["mcp" "fetch" "search" "web" "eio" "ocaml"]
+homepage: "https://github.com/mseri/snf-mcp"
+doc: "https://github.com/mseri/snf-mcp"
+bug-reports: "https://github.com/mseri/snf-mcp/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.17"}
+  "ca-certs"
+  "eio" {>= "0.13"}
+  "eio_main"
+  "fmt"
+  "cohttp"
+  "cohttp-eio"
+  "jsonrpc" {>= "1.7.0"}
+  "lambdasoup"
+  "logs"
+  "mirage-crypto-rng" {>= "1.2.0"}
+  "re"
+  "tls-eio"
+  "yojson" {< "3.0.0"}
```

Comment:
```suggestion
  "yojson" {>="1.6.0" & < "3.0.0"}
```
I can see `snf_mcp.0.2.0` had this lower bound - and that there's a lower bound `Yojson` failure on the CI :slightly_smiling_face: